### PR TITLE
Simple Container

### PR DESCRIPTION
**Pull request by benjdlambert**
### Problem
- Need a container to build `node` applications

### Solution
- Create a simple container that for now just exposes `npm`

### Notes
@stevenjack @revett

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+machine:
```
Comment (revett) [inline circle.yml]: Worth adding the `DOCKER_IMAGE` env var 👍 

https://github.com/vidsy/creator/blob/master/circle.yml#L2-L3

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM node:5.8.0-slim
+MAINTAINER charlie@vidsy.co
```
Comment (benjdlambert) [inline Dockerfile]: I don't have `blam@vidsy.co` so... 😒

**Comment by revett** [comment]
🍰 after comment

File: Dockerfile (hunk 2 above)
Comment (revett) [inline Dockerfile]: Use your own email 👍

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  override:
+    - docker build -f Dockerfile -t vidsyhq/node-builder .
+
+test:
+  override:
+    - echo "No tests here!"
+
+deployment:
+  hub:
+    branch: master
```
Comment (stevenjack) [inline circle.yml]: Worth taking a look at:

https://github.com/vidsy/assume-role/blob/master/circle.yml#L13-L20

and adopting the same for tagging the container releases..

**Comment by stevenjack** [comment]
🍰

**Pull request merged**